website/docs/error-reference.md

```diff
@@ -1126,6 +1126,26 @@ def publish_item(name, category = nil)
 end
 ```
 
+## 7009
+
+This error occurs when a value is used in place of a type:
```

Comment:
Given jez's comments, I think it would be good to add a note here that the example below is not exhaustive:

```suggestion
This error occurs when a value is used in place of a type.  There are many different situations where this can happen; one example is given below:
```